### PR TITLE
8289587: IllegalArgumentException: Color.rgb's red parameter (-16776961) expects color values 0-255

### PR DESCRIPTION
**Pull request by jperedadnr**
This PR solves an IAE exception that happens when invoking a ColorChooser control within a WebView, by getting the r, g, b values from the Color before calling the colorChooser.

A system test is included, to verify that there is no exception and the two native functions that are modified produce the expected color components.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Issue
 * [JDK-8289587](https://bugs.openjdk.org/browse/JDK-8289587): IllegalArgumentException: Color.rgb's red parameter (-16776961) expects color values 0-255

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Ambarish Rapte](https://openjdk.org/census#arapte) (@arapte - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx pull/809/head:pull/809` \
`$ git checkout pull/809`

Update a local copy of the PR: \
`$ git checkout pull/809` \
`$ git pull https://git.openjdk.org/jfx pull/809/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 809`

View PR using the GUI difftool: \
`$ git pr show -t 809`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/809.diff">https://git.openjdk.org/jfx/pull/809.diff</a>

</details>

### CODE REVIEW

**Comment by kevinrushforth** [comment]
I'll review this. @arapte or @jaybhaskar can one you also review?

**Review by kevinrushforth — APPROVED** [review]
Looks good. I confirm that the new test fails without the fix and passes with the fix. I see the construct `auto [r, g, b, a] = color.toSRGBALossy<uint8_t>()` is used in other places to similarly extract the color, so looks like it was just missed in the two places you found.

File: tests/system/src/test/java/test/javafx/scene/web/ColorChooserTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.paint.Color;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+import javafx.stage.Window;
+import org.junit.AfterClass;
```
Comment (mstr2) [inline tests/system/src/test/java/test/javafx/scene/web/ColorChooserTest.java:41]: Since this is a new file, you could also use the JUnit5 API.

**Review by arapte — APPROVED** [review]
Looks good.
Updated test fails without fix and passes with fix.

**Review by kevinrushforth — APPROVED** [review]

**Pull request closed without merging**